core/catalog-core-eventcommands/pom.xml

```diff
@@ -12,71 +12,115 @@
             <artifactId>catalog-core-commons</artifactId>
 
-		<!-- There is a conflict between geotools (specifically gt-xsd-filter) 
-			and jscience. Since commons-json does not directly depend on jscience, rahter 
-			it calls methods in commons-ddf that use jscience, we can add an exclusion 
-			stanza here to omit the transitive jscience dependency from the commons-ddf 
-			dependency and resolve its conflict with geotools. -->
-			<exclusions>
-				<exclusion>
-					<groupId>org.jscience</groupId>
-					<artifactId>jscience</artifactId>
-				</exclusion>
-			</exclusions>
-		</dependency>
+            <!-- There is a conflict between geotools (specifically gt-xsd-filter)
+                and jscience. Since commons-json does not directly depend on jscience, rahter
+                it calls methods in commons-ddf that use jscience, we can add an exclusion
+                stanza here to omit the transitive jscience dependency from the commons-ddf
+                dependency and resolve its conflict with geotools. -->
+            <exclusions>
+                <exclusion>
+                    <groupId>org.jscience</groupId>
+                    <artifactId>jscience</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
 
-	</dependencies>
-	
-	<build>
-		<plugins>
-		    <plugin>
-				<groupId>org.apache.felix</groupId>
-				<artifactId>maven-bundle-plugin</artifactId>
-				<extensions>true</extensions>
-				<configuration>
-					<instructions>
-						<Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-						<Export-Package />
-						<Import-Package>
-						    ddf.catalog.event,
-							org.apache.felix.service.command,
-							org.apache.felix.gogo.commands,
-							org.apache.karaf.shell.console,
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Export-Package/>
+                        <Import-Package>
+                            ddf.catalog.event,
+                            org.apache.felix.service.command,
+                            org.apache.felix.gogo.commands,
+                            org.apache.karaf.shell.console,
                             org.joda.time;version="[1.6.0,3.0.0)",
-							*
-						</Import-Package>
-					</instructions>
-				</configuration>
-			</plugin>
-		</plugins>
-	</build>
-	
-</project>
\ No newline at end of file
+                            *
+                        </Import-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
```

Comment:
Example from what I was talking about on the parent PR. Is this configuration already in the parent?